wp/plugin/wp.ml

```diff
@@ -186,6 +186,10 @@ let stack_size = Cmd.parameter Typ.(some int) "stack-size"
     ~doc:{|Sets the size of the stack, which should be denoted in bytes. By
            default, the size of the stack is 0x800000 which is 8MB.|}
 
+let user_func_spec = Cmd.parameter Typ.(some (t3 string string string)) "user-func-spec"
```

Comment:
Is this flag in the form of: sub name, precondition, postcondition? We might want to include this in the documentation.

Reply:
Good idea! I'll add that to the documentation so the syntax is clearly explained.